### DISCUSSION TITLE
A Thank You :D and a Deployed Product using Seq (starbeam.one)

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [datalust/seq-tickets#1667] [Show and tell]
A quick shout of appreciation real quickly the Seq team and particularly @nblumhardt for all their great assistance around Seq.  I am still a sole/individual developer so it's only me using your product, but it is still central to all my logging across 7 AppService instances in my Azure account + the 1 Container Instance to host the application responsible for logging.

FWIW I have Seq listed as a featured item in the digital products I sell.  You cannot see it unless you actually buy one (as of this morning [we've had 123 people do exactly that](https://alpha.starbeam.one/) 😁) but it's there trust me!

The application I have created is starbeam.one: https://alpha.starbeam.one

And an example product available for purchase can be seen here:
https://twitter.com/Mike_E_angelo/status/1579240848591814657/photo/4

For every digital product sold, I've got a list of featured brands in the `Featured` sub-tab there under the `Discovery` page and Seq is included.  While this particular product was released last night and doesn't show that featured list, I'll be sure to grab a screenshot of a product released tonight and get a better listing for you.

Anyways, I wanted to "show and tell" here.  Your support has been incredible and wanted to share my resulting appreciation.  I will be sure to declare my appreciation for your product in any way I can going forward. 👍

**Comment by Mike-E-angelo** [comment]
> I'll be sure to grab a screenshot of a product released tonight and get a better listing for you.

Well OK maybe "tomorrow morning" is better, but still. 😁

![image](https://user-images.githubusercontent.com/2931376/195082719-9c7a6e40-bd43-49dd-beb4-35950e7968d3.png)

Taken from the product ([available from here](https://alpha.starbeam.one/market/products/07cba3e8-2e50-4821-970e-08daaad7102a))

**Reply by nblumhardt** [reply]
Congrats on the launch, and thanks for the shout-out, @Mike-E-angelo :-)
